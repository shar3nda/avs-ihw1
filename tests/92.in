10
1 10 8 4 0 7 10 -1 8 10