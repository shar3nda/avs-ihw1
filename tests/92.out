0 7 10 -1 8 10